controllers/disruption_rollout_controller.go

```diff
@@ -241,6 +242,8 @@ func (r *DisruptionRolloutReconciler) handleTargetResourceMissingPastExpiration(
 		return fmt.Errorf("failed to delete instance: %w", err)
 	}
 
+	r.handleMetricSinkError(r.MetricsSink.MetricMissingTargetDeleted(DisruptionRolloutTags))
```

Comment:
should we have a different metric for the rollout? How do we differentiate rollout vs cron?

Reply:
It is a different metric, they dont share a prefix:
```

	metricPrefixRolloutController = "chaos.rollout.controller."
	metricPrefixCronController    = "chaos.cron.controller."
```